pkg/sysched/clientset/v1alpha1/profiles_test.go

```diff
@@ -0,0 +1,58 @@
+package v1alpha1
+
+import (
+	//"fmt"
+	"bytes"
+	"io/ioutil"
+	"encoding/json"
+	"testing"
+	"net/http"
+	"github.com/stretchr/testify/assert"
+	restfake "k8s.io/client-go/rest/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/security-profiles-operator/api/seccompprofile/v1beta1"
+)
+
+var spoResponse = map[string]interface{}{
```

Comment:
can you use this struct directly https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/api/seccompprofile/v1beta1/seccompprofile_types.go#L134 instead of using `map[string]inteface{}` and then marshall it to json? It'll be more readable.

Reply:
@PiotrProkop thanks for the suggestion! we did change the code to use the seccomp type but still had to keep the marshalling as we want to return it using the httpclient.